src/components/ForecastSummary.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Forecast from "./Forecast";
+import { getDay } from "./DataFunction";
+
+export default function Card({ userInput }) {
+  const [forecastData, setForecastData] = useState(null);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  useEffect(() => {
+    let timeoutId;
+
+    axios
```

Comment:
This should not really be in this component, you already have some logic fetching data from a different API in `App.js`, so why the duplication of work? Having the same kind of work being done in different places like this makes the code harder to understand as its unnecessarily complicated and is more prone to bugs as a result. This the data you need for the whole app could be gained in the App component and passed down, you should do this instead as it would keep things consistent